src/librssguard/core/messagesproxymodel.cpp

```diff
@@ -11,7 +11,7 @@
 #include <QTimer>
 
 MessagesProxyModel::MessagesProxyModel(MessagesModel* source_model, QObject* parent)
-  : QSortFilterProxyModel(parent), m_sourceModel(source_model), m_showUnreadOnly(false) {
+  : QSortFilterProxyModel(parent), m_sourceModel(source_model), m_filter(MessageFilter::NoFiltering) {
```

Comment:
I suggest renaming the enum to `MessageListFilter` as there already is `MessageFilter` class for a different feature.